packages/wdio-static-server-service/package.json

```diff
@@ -30,10 +30,13 @@
     "@wdio/logger": "6.4.7",
     "express": "^4.14.0",
     "fs-extra": "^9.0.0",
-    "morgan": "^1.7.0"
+    "morgan": "^1.7.0",
+    "@types/express": "^4.17.8",
```

Comment:
I think we can move types to dev dependencies

Reply:
Done

Reply:
Done